src/services/comparison.ts

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash';
+
+/**
+ * Order doesn't matter, converts to sets, deep compare via lodash isEqual
+ * @param arr1
+ * @param arr2
+ */
+
+export const areArraysEqual = (arr1: any[] | undefined, arr2: any[] | undefined) => {
```

Comment:
In general I would suggest to not accept undefined arguments, but since this is a very specific implementation to compare our state arrays, and simplifies the code on the other end, 👍 👍 👍